.github/workflows/tests.yaml

```diff
@@ -0,0 +1,52 @@
+name: Tests
+on:
+  push:
+    branches:
+      - master
+  pull_request:
+
+env:
+  COVERALLS_PARALLEL: true
+  LOGGING: info
+
+jobs:
+  core-tests:
+    runs-on: ubuntu-latest
+
+    strategy:
+      matrix:
+        config:
+          - {python-version: 3.5}
+          - {python-version: 3.6}
+          - {python-version: 3.7}
```

Comment:
Does bb support Python 3.8?

Reply:
Officially, no, but in practice, I'm pretty sure it should. I'm going to attempt to add support for both 3.8 and 3.9. :slightly_smiling_face: